src/main/java/org/verdictdb/core/querying/simplifier/ConsolidatedExecutionNode.java

```diff
@@ -234,11 +234,13 @@ public ExecutionInfoToken createToken(DbmsQueryResult result) {
     // DbmsQueryResultMetaData should be false for all columns.
     if (token.containsKey("queryResult")) {
       DbmsQueryResult queryResult = (DbmsQueryResult) token.getValue("queryResult");
-      List<Boolean> isAggregate = new ArrayList<>();
-      for (int i=0;i<queryResult.getColumnCount();i++) {
-        isAggregate.add(false);
+      if (queryResult.getColumnCount() != queryResult.getMetaData().isAggregate.size()) {
```

Comment:
Can this logic handle the case like, "select col1, * from lineitem"?

Reply:
I haven't thought this case. But I think the answer is yes if the query is "select col1, * from lineitem". I think it might be problematic when the query is "select sum(price), * from lineitem group by something". I think I should change the logic slightly to fix this.

Reply:
It would be nice if the column information (for filling in the metadata for asterisk) can be obtained from the table names in the from list (of the query).

Reply:
I think an easier approach is to compare the column information from queryResult metadata with isAggregate field to find what asterisk column contains.

Reply:
That sounds good.